### PR TITLE
make v3 default for all web components

### PR DESCRIPTION
**Pull request by it-harrison**
## Chromatic

https://2405-iii--65a6e2ed2314f7b8f98609d8.chromatic.com

## Description
This PR sets the default value for the `uswds` prop to true for all components with a v3 variant. It also makes adjustments to components that used composition with other components so that the correct version is used. Finally, it updates stories and moves up v3 stories to the top of the sidebar in Storybook.

Closes [2405](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2405)

## QA Checklist
- [ ] Component maintains 1:1 parity with design mocks
- [ ] Text is consistent with what's been provided in the mocks
- [ ] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ ] Tab order and focus state work as expected
- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ ] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ ] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Acceptance criteria
- [ ] QA checklist has been completed
- [ ] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-button/va-button.tsx [hunk 1]
```diff
@@ -87,6 +90,14 @@ export class VaButton {
   })
   componentLibraryAnalytics: EventEmitter;
 
+  componentDidLoad() {
+    // check if the element has a class named uswds-false added from parent
+    if (this.el.classList.contains('uswds-false')) {
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-button/va-button.tsx:95]: @it-harrison Do you mind adding some details here as a PR comment about why you're needing to do this?

Comment (it-harrison) [reply packages/web-components/src/components/va-button/va-button.tsx:95]: @jamigibbs sure - there is a bug with Stencil in which an attribute set on a child component inside a parent does not get added to the rendered HTML. For instance, if we want the v1 version of `va-button-pair` and we set `uswds="false"` on `VaButtonPair` then, although the `uswds` prop is added to the child `va-button` components inside `va-button-pair`, when rendered the `va-button` elements will not have `uswds="false"`. So it is necessary to add this attribute manually. When all v1 components have been decommissioned we can remove this code where it occurs.

**Comment by jamigibbs** [comment]
Any idea why this component is hanging out at the bottom of the Storybook sidebar?

Screenshot 2024-03-11 at 10 51 30 AM

**Comment by it-harrison** [comment]
> Any idea why this component is hanging out at the bottom of the Storybook sidebar?
> 
> Screenshot 2024-03-11 at 10 51 30 AM

@jamigibbs there was a v1 version of statement-of-truth that should not be included - I removed it and tweaked the v3 story so it should be ok now

**Review by jamigibbs — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Pull request merged**